pkg/version/version_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2016 CNI authors
```

Comment:
New file, so it probably should be 2018 ;)

Reply:
@jellonek fixed